dnf5/context.cpp

```diff
@@ -251,7 +255,7 @@ void Context::Impl::apply_repository_setopts() {
                 repo->get_config().opt_binds().at(key).new_string(
                     libdnf5::Option::Priority::COMMANDLINE, setopt.second);
             } catch (const std::exception & ex) {
-                std::cout << "setopt: \"" + setopt.first + "." + setopt.second + "\": " + ex.what() << std::endl;
+                std::cerr << "setopt: \"" + setopt.first + "." + setopt.second + "\": " + ex.what() << std::endl;
```

Comment:
Here, we should use the `err_stream` reference, right? And probably in other members of `Context::Impl` as well...

Reply:
Done.